addOns/pscanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -25,6 +25,7 @@ <H2>Base64 Disclosure</H2>
 <strong>Note:</strong> At Low Threshold all occurrences within each response will be included.
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/Base64Disclosure.java">Base64Disclosure.java</a>
+Alert ID: <a href="https://www.zaproxy.org/docs/alerts/10094">10094</a>.
```

Comment:
The suggestion in https://github.com/zaproxy/zaproxy/issues/8189 was to have this immediatelly after the alert heading, but I'm ok with it here too :grin:
However this is HTML so it will need a `<br>` on the previous line otherwise it will be on the same line as "Latest code.." which I dont think will look right.